### PR TITLE
Added BMI command

### LINKED ISSUE #875: BMI calculator

### PR DESCRIPTION
**Pull request by upedd**
Closes #875

### CODE REVIEW

File: src/commands/misc/bmi.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    color: '#FF5722',
+    min: 40,
+    max: Infinity
+  }
+}
+
+module.exports = class BMI extends Command {
+  constructor (client) {
+    super({
+      name: 'bmi',
```
Comment (almeidx) [inline src/commands/misc/bmi.js:58]: maybe add a `bobymassindex` alias?

**Comment by upedd** [comment]
Done

File: src/locales/en-US/commands.json [hunk 2]
```diff
@@ -1775,5 +1775,22 @@
     "nis": "NIS",
     "amount": "Amount",
     "city": "City"
+  },
+  "bmi": {
+    "commandDescription": "Calculates your BMI",
+    "commandUsage": "<weigth in kilograms> <height in meters>",
+    "missingWeight": "You have to specify a valid weight in kilograms.",
+    "missingHeight": "You have to specify a valid weight in kilograms.",
```
Comment (almeidx) [inline src/locales/en-US/commands.json:1783]:
```suggestion
    "missingHeight": "You have to specify a valid height in kilograms.",
```

**Review by almeidx — CHANGES_REQUESTED** [review]

**Review by almeidx — DISMISSED** [review]

**Review by pedrofracassi — DISMISSED** [review]
LGTM

**Comment by Doges** [comment]
Please resolve the conflicts.

**Review by almeidx — APPROVED** [review]
LGTM

**Review by Doges — APPROVED** [review]

**Pull request merged**